frontend/plan/src/components/schedule/Stats.js

```diff
@@ -32,15 +32,15 @@ class Stats extends Component {
         const { meetings } = this.props;
         let earliestStart;
         let latestEnd;
-        let minHoursADay;
+        let totalCUs;
         let maxHoursADay;
         let totalHours;
         let averageHours;
         let avgs = {};
         if (meetings.length === 0) {
             earliestStart = "—";
             latestEnd = "—";
-            minHoursADay = "—";
+            totalCUs = "–";
```

Comment:
I think this should be an em dash (option-dash) instead of a regular dash:
<img width="149" alt="Screen Shot 2019-10-22 at 1 40 28 PM" src="https://user-images.githubusercontent.com/436045/67313782-8e0bf000-f4d1-11e9-9ec3-adaf9c07d7bb.png">